sklearn/linear_model/tests/test_ridge.py

```diff
@@ -487,6 +487,35 @@ def test_class_weights():
     assert_array_almost_equal(clf.intercept_, clfa.intercept_)
 
 
+def test_class_weight_vs_sample_weight():
```

Comment:
True... as long as we don't have any classification-oriented transformers
that are not predictors that accept sample and class weights!

On 11 June 2015 at 03:46, Andreas Mueller <notifications@github.com> wrote:

> In sklearn/linear_model/tests/test_ridge.py
> <https://github.com/scikit-learn/scikit-learn/pull/4838#discussion_r32145069>
> :
>
> > @@ -487,6 +487,35 @@ def test_class_weights():
> >      assert_array_almost_equal(clf.intercept_, clfa.intercept_)
> >
> >
> > +def test_class_weight_vs_sample_weight():
>
> Adding one that checks if sample_weights is equivalent to doing the
> corresponding class_weights would be nice, too. I feel it should be
> pretty straight-forward, though. Why would you need a helper? Just check if
> decision_function is almost_equal.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scikit-learn/scikit-learn/pull/4838/files#r32145069>.
>